app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java

```diff
@@ -228,35 +229,64 @@ void setTitleDescButton() {
      */
     @OnTouch(R.id.titleEdit)
     boolean titleInfo(View view, MotionEvent motionEvent) {
-        //Should replace right with end to support different right-to-left languages as well
-        final int value = titleEdit.getRight() - titleEdit.getCompoundDrawables()[2].getBounds().width();
-
-        if (motionEvent.getAction() == ACTION_UP && motionEvent.getRawX() >= value) {
-            new AlertDialog.Builder(getContext())
-                    .setTitle(R.string.media_detail_title)
-                    .setMessage(R.string.title_info)
-                    .setCancelable(true)
-                    .setNeutralButton(android.R.string.ok, (dialog, id) -> dialog.cancel())
-                    .create()
-                    .show();
-            return true;
+        final int value;
+        if (ViewCompat.getLayoutDirection(getView()) == ViewCompat.LAYOUT_DIRECTION_LTR) {
+            value = titleEdit.getRight() - titleEdit.getCompoundDrawables()[2].getBounds().width();
+            if (motionEvent.getAction() == ACTION_UP && motionEvent.getRawX() >= value) {
+                new AlertDialog.Builder(getContext())
```

Comment:
the logic for creating a alert dialog is duplicated. It can be extracted out to a function.

Reply:
That was careless of me, I overlooked it. I'll correct it and push immediately.